src/components/kompas-paywall-video/kompas-paywall-video.tsx

```diff
@@ -102,6 +102,7 @@ export class KompasPaywallVideo {
       paywall_subscription_id: this.paywall_subscription_id,
       paywall_subscription_price: this.paywall_subscription_price,
       paywall_position: this.paywall_position,
+      content_categories: this.tracker_content_category,
```

Comment:
nama paramnya minta tolong di sesuain 
```suggestion
      content_categories: this.tracker_content_categories,
```